mirroroperator/registrymirror.py

```diff
@@ -102,8 +101,10 @@ def __init__(self, event_type, namespace, hostess_docker_registry,
                 limit_except HEAD GET OPTIONS {{{{
                     deny all;
                 }}}}
-
-                proxy_pass                    https://{upstream_fqdn};
+                
+                resolver 
```

Comment:
do you not require a value here? 

https://nginx.org/en/docs/http/ngx_http_core_module.html#resolver

Reply:
this doc 
https://tenzer.dk/nginx-with-dynamic-upstreams/

suggests that the `resolver` and `set` lines should be outside of the location block, I'm not sure if that matters

Reply:
resolver is a placeholder. The value is dynamically set in the generate_stateful_set function. 

There doesn't seem to be a definitive but will move it to be outside the location block